script/Ciel/Day 7/07_01_CIEL07_1.txt

```diff
@@ -347,44 +359,45 @@ Perhaps the only unusual thing here is the sign hanging in front of the departme
 [sha:29adf0bf31be52fc531d533ae9479cc11266add5]{
 -- Page 26, Offset 14868.
 -- 先輩たちが人知れず片付けてくれたのだろう。
-I suppose Senpai and her people cleaned it up without anything getting out.
+I guess Senpai and the other members of the Church cleaned the place up without anyone finding out what happened.
 }
 [sha:fbb0c9af9f0e671958c965b144ed2518a816e3ba]{
 -- Page 26, Offset 14869.
 -- 吸血鬼のねぐらに通じていたデパート。
-This was the department store which lead into the vampire lair.
+That's the department store which used to lead to the vampire's lair.
 }
 [sha:d897186c5ee06bf5d910485b1cc5e941f1e727d6]{
 -- Page 26, Offset 14870.
 -- 地下墓地が倒壊し、ヴローヴが死んだ以上、あのデパートもただの廃墟になったのだ。
-Now that Vlov is dead, and the catacombs have collapsed, it has turned into nothing but ruins.
+// Building itself seems fine in the CG, leaning more towards abandoned for 廃墟 than ruined
```

Comment:
I think it's going for a poetic "remains of a building" kinda deal but i can't think of anything en to match. No need for a change because this line is fine, but if you've got anything i think it would improve the tl